.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
 /dist
+.idea
```

Comment:
This is a local environment [thing that you can do for your whole machine](https://stackoverflow.com/questions/7335420/global-git-ignore).